src/charts/bubble-chart.js

```diff
@@ -464,6 +464,17 @@ export default function bubbleChart(parent, chartGroup) {
       const radiusAccessor = _chart.radiusValueAccessor()
       data.sort((a, b) => d3.descending(radiusAccessor(a), radiusAccessor(b)))
     }
+
+    const domain = data.map(d => d.key0)
+    const range = data.map((d, i) => _chart.getColor(d, i))
+    if (
+      _chart.customDomainRangeActive() ||
```

Comment:
It doesn't seem like this is set anywhere after being initialized to false, can it just be removed?